evaluation/tasks/mrpc-confirmation/mrpc-confirmation.py

```diff
@@ -0,0 +1,152 @@
+        self.items = []
+
+        for sample in mrpc:
+            # detokenize the text, since MRPC is tokenized
+            # with MosesDetokenizer('en') as detokenize:
+            #    sample["sentence1"] = detokenize(sample["sentence1"].split())
+            # print(sample["sentence1"])
+            prompt = TEMPLATE_PARAPHRASE.render(
+                sent1=sample["sentence1"],
+            )
+
+            # Tokenize and construct this sample
+            inputs = tokenizer(
+                prompt,
+                return_tensors="pt",
+                truncation=True,
+            )
+            self.items.append(
+                {
+                    "prompt": prompt,
+                    "sentence1": sample["sentence1"],
+                    "input_ids": inputs["input_ids"],
+                    "attention_mask": inputs["attention_mask"],
+                    "input_len": inputs["attention_mask"].shape[1],
+                }
+            )
+
+    def __len__(self):
+        return len(self.items)
+
+    def __getitem__(self, index):
+        return self.items[index]
+
+
+class MRPCNegativeTask(AutoTask):
+    @staticmethod
+    def get_display_name() -> str:
+        return "mrpc-confirmation"
+
+    def evaluate(self) -> None:
+        dataset = MRPCDataset(self.tokenizer)
+
+        accuracy = 0
+        consistency = 0
+
+        paraphrase_prompts = []
+        confirmation_prompts = []
+        paraphrase_prompt_answers = []
+        confirmation_prompt_answers = []
+
+        is_first = True
+
+        logger = get_logger()
+        for sample in tqdm(dataset, desc=f"Evaluating {self.get_display_name()}"):
+            def get_output(sample):
+                with torch.no_grad():
+                    output = self.model.generate(
+                        input_ids=sample["input_ids"].to(self.device),
+                        attention_mask=sample["attention_mask"].to(self.device),
+                        max_length=min(sample["input_len"] * 2, 1024),
+                        # hard-coded to 1024 since each model has diferent naming for max length
+                        min_length=self.args.min_length,
+                        do_sample=self.args.do_sample,  # need to be set to true not to use greedy sampling
+                        early_stopping=self.args.early_stopping,
+                        # whether to stop when num_beams sentences are generated
+                        num_beams=self.args.num_beams,
+                        temperature=self.args.temperature,  # lower than 1 conservative, greater than one diverse
+                        top_k=self.args.top_k,
+                        # number of highest probability vocabulary tokens to keep for top-k-filtering
+                        top_p=self.args.top_p,  #
+                        repetition_penalty=self.args.repetition_penalty,
+                        length_penalty=self.args.length_penalty  # 1 no penalty >1 foster long sentences
```

Comment:
Can we refactor so that all the comment are consistently above the the argument line? e.g. 
```
# need to be set to true not to use greedy sampling
do_sample=self.args.do_sample, 
```